### DISCUSSION TITLE
How to disable an app?

### DISCUSSION THREAD

**Discussion by abhi-k9** [Hamza417/Inure#365] [Q&A]
I am unable to find the option to disable an app.

![Screenshot_20240905_092655_Inure](https://github.com/user-attachments/assets/ed6e0e1f-0141-4337-8d4c-5f54acfd9e57)

I have a few apps disabled, so it should be possible to do so.
![Screenshot_20240905_092629_Inure](https://github.com/user-attachments/assets/5bbcbd6b-0ffd-42c0-9e4f-7d6927eeaea9)

Thanks.

**Comment by Hamza417** [comment]
_The question has been answered in #364._
